addons/confirma/src/formatters/CollectionFormatter.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using System.Collections.Generic;
+using Confirma.Helpers;
+
+namespace Confirma.Formatters;
+
+public class CollectionFormatter : Formatter
+{
+    public override string Format(object value)
+    {
+        string result = "null";
+
+        ICollection<object?>? c = value as ICollection<object?>;
+        if (c is not null)
+        {
+            return CollectionHelper.ToString(c);
+        }
+        else if (value is Array a)
+        {
+            result = CollectionHelper.ToString((ICollection<object?>)a);
+        }
+        else
+        {
+            return new DefaultFormatter().Format(value);
+        }
+
+        return $"[{result}]";
```

Comment:
Lines 27 and 11 are only used by the second if statement, so they can be moved to one place
```suggestion
        ICollection<object?>? c = value as ICollection<object?>;
        if (c is not null)
        {
            return CollectionHelper.ToString(c);
        }
        else if (value is Array a)
        {
            string result = CollectionHelper.ToString((ICollection<object?>)a);
            return $"[{result}]";
        }
        else
        {
            return new DefaultFormatter().Format(value);
        }
```